apis_core/apis_entities/views.py

```diff
@@ -183,7 +183,7 @@ def get_table(self, **kwargs):
             "columns"
         )  # populates "Select additional columns" dropdown
         try:
-            default_cols = self.entity_settings["table_fields"]
+            default_cols = settings.APIS_ENTITIES.get(class_name)["table_fields"]
```

Comment:
here the settings were again replaced for the ones from models.py. given that in models.py we use either the one or the other to set the settings, it should read here from the settings rather than the models.py